Userland/Libraries/LibELF/DynamicObject.cpp

```diff
@@ -176,6 +176,14 @@ void DynamicObject::parse()
             break;
         case DT_SYMBOLIC:
             break;
+        case DT_VERSYM:
+        case DT_VERNEEDED:
+        case DT_VERNEEDEDNUM:
+        case DT_VERDEF:
+        case DT_VERDEFNUM:
+            // NOTE: We simply ignore these tags (which are usually found in ported software) 
+            // as we don't care about library backwards-compatibility at all.
+            break;
```

Comment:
While we don't care about them, Daniel said that we should still warn in case these tags end up in our own code. Therefore, we should probably keep the warnings for now, they don't break anything either way.